site/_docs/history.md

```diff
@@ -57,6 +57,9 @@ other software versions as specified in gradle.properties.
 #### Bug-fixes, API changes and minor enhancements
 {: #fixes-1-37-0}
 
+* The visibility of the method SqlCall.getCallSignature has been converted
```

Comment:
Thanks for including this. I'd suggest to give a bit more info, e.g. something a long the lines:

In the context of  [<a href="https://issues.apache.org/jira/browse/CALCITE-6015">CALCITE-6015</a>] the visibility of the method `SqlCall.getCallSignature` has been converted from `protected` to `public`, so any subclass overriding it might need to be adjusted accordingly.

Reply:
@rubenada changed List to Set and updated the history file.